README.md

```diff
@@ -0,0 +1,37 @@
+# ElasticSearch on Rancher
+
+> Rancher configuration for running an Elasticsearch cluster
+
+## Services deployed:
+
+- `es-master`: Elasticsearch nodes, part of the Elasticsearch cluster, one runs on every host that has an `escluster=true` tag defined.
+- Justwatch [Exporter]: Exposes Prometheus metrics on the cluster status.
+- es-client: Rancher cattle DNS service, which points the `es-client` domain name to `es-master`, for backwards compatibility with existing services.
+
+## Deployment
+
+Deployment is driven by Travis (Dev and PR review deployments) and TeamCity (staging and production deployments). We use the NHS.UK [CI-Deployment] repo for deployment to the Rancher environment, this sets the following ENV variables.
+
+## Environment variables
+
+| Variable | Description | Default | 
+| -------- | ----------- | ------- |
+| `ES_MIN_MASTER_NODES` | The minimum nodes required to run the cluster | `2` |
+| `ES_DATA_VOL` | The name of the local volume on the host to use | `esdata` |
```

Comment:
I recall that there was a rational for specifying a data volume to do with persistence of data - there is some discussion of this in Slack. Could this be captured here somewhere or maybe even in an ADR.

Reply:
I think it's implied that there's a need for a data volume here. That probably should be captured in an ADR.